### PR TITLE
Fix/iteration2 script refactor

### LINKED ISSUE #10: Iteration 2: Display how much water user consumed today
A display to show how much water they have consumed today (these displays are often called “widgets” in the FE tech world)

### LINKED ISSUE #11: Iteration 2: Display how much water user has consumed in last week
A display to show much water they have consumed each day over the course of the latest week

### PR DESCRIPTION
**Pull request by Shakikka**
## What does this PR do?

- Adds API capability and display for initial hydration information.
- Refactors scripts.js to accommodate these changes.

## How should it be tested in the UI?

- Run the API, and the server, and then look at the 2 middle columns of information.

### CODE REVIEW

**Comment by Shakikka** [comment]
[Add Fetch API for Iteration 2](https://trello.com/c/CIkLxAsQ/36-add-fetch-api-for-iteration-2)

**Comment by Shakikka** [comment]
[Refactor scripts.js *Iteration 2*](https://trello.com/c/2d8CGujy/33-refactor-scriptsjs-iteration-2)

File: src/scripts.js [hunk 1]
```diff
@@ -165,12 +149,12 @@ function makeRandomDate(userStorage, id, dataSet) {
...
-  hydrationThisWeek.insertAdjacentHTML('afterBegin', makeHydrationHTML(id, hydrationInfo, userStorage, hydrationInfo.calculateFirstWeekOunces(userStorage, id)));
-  hydrationEarlierWeek.insertAdjacentHTML('afterBegin', makeHydrationHTML(id, hydrationInfo, userStorage, hydrationInfo.calculateRandomWeekOunces(laterDateString, id, userStorage)));
-}
+// function addHydrationInfo(id, hydrationInfo, dateString, userStorage, laterDateString) {
+//   hydrationToday.insertAdjacentHTML('afterBegin', `<p>You drank</p><p><span class="number">${hydrationInfo.calculateDailyOunces(id, dateString)}</span></p><p>oz water today.</p>`);
+//   hydrationAverage.insertAdjacentHTML('afterBegin', `<p>Your average water intake is</p><p><span class="number">${hydrationInfo.calculateAverageOunces(id)}</span></p> <p>oz per day.</p>`)
+//   hydrationThisWeek.insertAdjacentHTML('afterBegin', makeHydrationHTML(id, hydrationInfo, userStorage, hydrationInfo.calculateFirstWeekOunces(userStorage, id)));
+//   hydrationEarlierWeek.insertAdjacentHTML('afterBegin', makeHydrationHTML(id, hydrationInfo, userStorage, hydrationInfo.calculateRandomWeekOunces(laterDateString, id, userStorage)));
+// }
```
Comment (m1073496) [inline src/scripts.js:157]: glad to see this go, nice work

File: src/scripts.js [hunk 2]
```diff
@@ -53,47 +53,39 @@ const bestUserSteps = document.getElementById('bestUserSteps');
...
+  displayHydrationInfo(userRepo, userNowId, hydrationInfo)
+}
+
+function displayHydrationInfo(userInfo, id, hydrationInfo) {
+  let hydrationRepo = new Hydration(hydrationInfo);
+  let today = makeToday(userInfo, id, hydrationRepo.hydrationData)
+  hydrationToday.insertAdjacentHTML('afterBegin', `<p>You drank</p><p><span
+  class="number">${hydrationRepo.calculateDailyOunces(id, today)}</span></p><p>oz water today.</p>`);
+  hydrationThisWeek.insertAdjacentHTML('afterBegin', makeHydrationHTML(id,
+    hydrationRepo, userInfo, hydrationRepo.calculateFirstWeekOunces(userInfo, id)))
```
Comment (m1073496) [inline src/scripts.js:88]: NIce

**Review by m1073496 — APPROVED** [review]

**Pull request merged**